source/_docs/automation/using_blueprints.markdown

```diff
@@ -44,3 +44,7 @@ The Home Assistant Community forums have a specific tag for blueprints. This tag
 [Visit the Home Assistant forums][blueprint-forums]
 
 [blueprint-forums]: /get-blueprints
+
+## Troubleshooting missing automations
+
+When creating automations using blueprints, you may find they appear to be created but never show in the automations page. When creating automations using the GUI, they are written to `automations.yaml` even if you have a [split configuration](/docs/configuration/splitting_configuration/#example-combine-include_dir_merge_list-with-automationsyaml). Make sure you are correctly including the base `automations.yaml` file to ensure they are loaded.
```

Comment:
We shouldn't mention split configuration at all. We should just mention that if new automations are not showing up, users need to ensure that they add `automation: !include automations.yaml` to their `configuration.yaml`. 

We should consider also updating the default written config to include this as a comment.

Reply:
So - in all of this, I don't mind the direction - just that I had a split config from a year ago - which didn't include the default `automations.yaml` - and I couldn't work out why blueprints didn't work.

And offering an option/opinion on the change.